File: Front/ProEventos.APP/src/app/models/Evento.ts
import { Lote } from "./Lote"
import { Palestrante } from "./Palestrante";
import { RedeSocial } from "./RedeSocial"

export interface Evento {

   id: number;
   local: string;
   dataEvento?: Date;
   tema: string;
   qtdPessoas: number;
   lotes: Lote[];
   imagemUrl: string;
   telefone: string;
   redesSociais: RedeSocial[];
   PalestrantesEventos: Palestrante[];
}

File: Back/src/ProEventos.Domain/Models/Evento.cs
using System.ComponentModel.DataAnnotations.Schema;
using ProEventos.Domain.Identity;

namespace ProEventos.Domain
{
    [Table("eventos")]
    public class Evento
    {
        public int Id {get;set;}
        public string Local {get;set;}
        public DateTime? DataEvento {get;set;}
        public string Tema {get;set;}
        public int QtdPessoas {get;set;}
        public string ImagemUrl {get;set;}
        public string Email {get; set;}
        public string Telefone {get;set;}
        public int UserId { get; set; }
        public User User { get; set; }
        public IEnumerable<Lote> Lotes {get;set;}
        public IEnumerable<RedeSocial> RedesSociais {get; set;}
        public IEnumerable<PalestranteEvento> PalestrantesEventos { get; set; }
    }
}